modules/social_features/social_profile/social_profile.module

```diff
@@ -543,8 +543,8 @@ function social_profile_form_user_form_alter(&$form, FormStateInterface $form_st
       ],
     ];
     $is_multilingual = \Drupal::languageManager()->isMultilingual();
+    $user_data = \Drupal::service('user.data');
     if ($is_multilingual) {
-      $user_data = \Drupal::service('user.data');
```

Comment:
Please remove this change from this PR. I understand it's slightly cleaner but it's unrelated and causes a merge conflict with another work in progress PR that'll remove this altogether :D

Reply:
Sure.